### DISCUSSION TITLE
How can I see retrieve my transaction.

### DISCUSSION THREAD

**Discussion by TheBullFerdinand** [blockscout/blockscout#9557] [Support]
0xbe13af81a20992ed53adcdf010352711b8a64eb736d57172b4f0ff84dcdf6f97

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
